### PR TITLE
[docs] Document structure of `META.yml` files

### LINKED ISSUE #12199: Document META.yml
The structure of these files is currently undocumented and probably should be.

cc/ @foolip

### PR DESCRIPTION
**Pull request by jugglinmike**
Resolves gh-12199

### CODE REVIEW

**Review by sideshowbarker — APPROVED** [review]

File: docs/reviewing-tests/index.md [hunk 1]
```diff
@@ -46,10 +46,10 @@ collaborator before it can be merged.
 
 ## Notifications
 
-META.yml files are used only to indicate who should be notified of pull
+META.yml files are used to indicate who should be notified of pull
 requests.  If you are interested in receiving notifications of proposed
-changes to tests in a given directory, feel free to add yourself to the
-META.yml file.
+changes to tests in a given directory, feel free to add your GitHub account
+name to the `suggested_reviewers` list in the META.yml file.
```
Comment (gsnedders) [inline docs/reviewing-tests/index.md]:
```suggestion
username to the `suggested_reviewers` list in the META.yml file.
```

File: docs/test-suite-design.md [hunk 2]
```diff
@@ -32,6 +32,13 @@ For example, tests in HTML for ["The History
 interface"](https://html.spec.whatwg.org/multipage/history.html#the-history-interface)
 are located in `html/browsers/history/the-history-interface/`.
 
+Many directories also include a file named `META.yml`. This file may define any
+of the following properties:
+
+- `spec` - a link to the specification that the tests in the directory exercise
+- `suggested_reviewers` - a list of GitHub account names belonging to people
```
Comment (gsnedders) [inline docs/test-suite-design.md]:
```suggestion
- `suggested_reviewers` - a list of GitHub account username belonging to people
```

File: docs/test-suite-design.md [hunk 3]
```diff
@@ -32,6 +32,13 @@ For example, tests in HTML for ["The History
 interface"](https://html.spec.whatwg.org/multipage/history.html#the-history-interface)
 are located in `html/browsers/history/the-history-interface/`.
 
+Many directories also include a file named `META.yml`. This file may define any
+of the following properties:
+
+- `spec` - a link to the specification that the tests in the directory exercise
```
Comment (gsnedders) [inline docs/test-suite-design.md]: I somewhat dislike this use of the word "exercise"—it seems just obscure enough to be a bit confusing; or maybe that's just the grammar here with it following the object?

Comment (jgraham) [reply docs/test-suite-design.md]: Yeah, I think `specification exercised by the tests in the directory` is fine, although `covered` would be even better.

File: docs/test-suite-design.md [hunk 4]
```diff
@@ -32,6 +32,13 @@ For example, tests in HTML for ["The History
 interface"](https://html.spec.whatwg.org/multipage/history.html#the-history-interface)
 are located in `html/browsers/history/the-history-interface/`.
 
+Many directories also include a file named `META.yml`. This file may define any
+of the following properties:
+
+- `spec` - a link to the specification covered by the tests in the directory
+- `suggested_reviewers` - a list of GitHub account namesuser belonging to
```
Comment (gsnedders) [inline docs/test-suite-design.md]:
```suggestion
- `suggested_reviewers` - a list of GitHub account username belonging to
```

**Review by gsnedders — APPROVED** [review]

**Pull request merged**